pkg/config/datafileprojectconfig/mappers/experiment_test.go

```diff
@@ -114,6 +117,53 @@ func TestMapExperiments(t *testing.T) {
 	assert.Equal(t, expectedExperimentKeyMap, experimentKeyMap)
 }
 
+func TestMergeExperiments(t *testing.T) {
+	const testExperimentString = `{
+		"id": "11111",
+	}`
+
+	const testGroupString = `{
+		"policy": "random",
+      "trafficAllocation": [
+        {
+          "entityId": "21113",
+          "endOfRange": 7000
+        },
+        {
+          "entityId": "21114",
+          "endOfRange": 10000
+        }
+	  ],
+	  "experiments": [
+		  {
+			"id": "11112"
+		  }
+	  ],
+	  "id":"11112"
+	}`
+
+	var rawExperiment datafileEntities.Experiment
+	var rawGroup datafileEntities.Group
+	var json = jsoniter.ConfigCompatibleWithStandardLibrary
+	json.Unmarshal([]byte(testExperimentString), &rawExperiment)
```

Comment:
Sorry I didn't actually catch this earlier, but we shouldn't need to unmarshal the json string into the raw entities. We should just directly create a `datafileEntities.Experiment{}` with the properties that we care about

Reply:
Thanks, done.